### PR TITLE
added max

### LINKED ISSUE #5836: max

### PR DESCRIPTION
**Pull request by Viditagarwal7479**
Close #5836

### CODE REVIEW

File: ivy/functional/frontends/torch/Tensor.py [hunk 1]
```diff
@@ -66,7 +66,10 @@ def amax(self, dim=None, keepdim=False):
     def contiguous(self, memory_format=torch.contiguous_format):
         return self.data
 
-    # Special Methoods #
+    def max(self, dim=None, keepdim=False):
+        return torch_frontend.max(self.data, dim=dim, keepdim=keepdim)
```
Comment (zaeemansari70) [inline ivy/functional/frontends/torch/Tensor.py:70]: It's giving an error: `AttributeError: module 'ivy.functional.frontends.torch' has no attribute 'max'`
Make sure that the ordinary `max()` function is implemented before implementing the instance method.
You can implement that If you want. 🙂

**Review by zaeemansari70 — COMMENTED** [review]
Hey! 
First of all thank you soo much for contributing to ivy! 🙂
Your PR looks amazing ✨!
Actually, you have implemented the instance method of a function whose ordinary method is not yet implemented 🙂
I have mentioned the exact error in the comments as well.
Thanks!

File: ivy/functional/frontends/torch/Tensor.py (hunk 1 above)
Comment (zaeemansari70) [inline ivy/functional/frontends/torch/Tensor.py:70]: Hi 🙂
The torch frontend doesn't have a `max()` function yet.
This is an instance method that you've implemented, which should call the ordinary `max()` function for torch frontend which is not implemented yet.
 
We'd love if you'd implement the ordinary function too, as every contribution here at Ivy is appreciated.
Thanks! 🙂

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]

**Comment by Viditagarwal7479** [comment]
Hey, ivy.max() and all needed functions were already there I noticed a sort of linking issue which I fixed and hope it works well now.:smile:

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
I've requested a few more changes!🙂:

1. For the completion on one task, one has to implement a function and it's test, this makes a one single complete task.
2. The instance and the special methods can not be implemented if the ordinary function is not yet completely implemented  which is ` max()` in this case.

You've just added the `max()` function in a file. 
What I would suggest in this case is, make a separate PR, where you implement the `max()` function in `ivy/functional/frontends/torch/reduction_ops.py` and it's test in `ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py`.
You can revert your change in `ivy/functional/frontends/torch/pointwise_ops.py`, Once your PR for the original function is merged, we will look into this PR.

Do let me know if you have any other questions.
Thanks! 🙂
You can assign the function form here https://github.com/unifyai/ivy/issues/13

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
As the original torch frontend `max()` function is now implemented https://github.com/unifyai/ivy/issues/13, your instance method will now be working. 
I've requested a few changes.🙂
Thanks! 🙂

**Review by zaeemansari70 — APPROVED** [review]
Thank you soo much for contributing to ivy! 🙂
Your changes looks good to be merged now!
I hope you enjoyed contributing to Ivy, looking forward to your contribution for Ivy in the future too 🙂
Thanks!

**Pull request merged**